content/docs/setup/kubernetes/spec-requirements/index.md

```diff
@@ -38,3 +38,47 @@ cluster must satisfy the following requirements:
 
 * _**`NET_ADMIN` capability**:_ If pod security policies are enforced in your cluster and unless you use the [Istio CNI Plugin](/docs/setup/kubernetes/istio-cni-install/), your pods must have the `NET_ADMIN` capability allowed.
 See [Required Pod Capabilities](/help/ops/setup/required-pod-capabilities/).
+
+* _**Service Ports and Protocols**_: Ensure that your service's ports and protocols are available and don't conflict with the [Istio used ports and protocols](#istio-used-ports) for the Istio components be used.
+
+## Istio Used Ports
```

Comment:
We don't capitalize headings.

How about "Istio port usage" instead?

Reply:
This is one of the few instances where I would use passive voice to avoid a weird sentence and to emphasize "Ports":
```suggestion
## Ports used by Istio
```

Reply:
Done (along with references)